spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/Util.scala

```diff
@@ -134,6 +136,84 @@ object Util {
     }
   }
 
+  private[bigdl] def getAndClearConsts[T: ClassTag](
+        model: Container[_, _, T])(implicit ev: TensorNumeric[T]): Map[String, Tensor[_]] = {
+    val moduleConsts = model.findModules("Const")
+      .map(_.asInstanceOf[Const[T, _]])
+      .map(v => (v, newTensor(v.value)))
```

Comment:
why not use clone tensor?

Reply:
Reuse the storage array to prevent memory copy and GC.